src/patternfly/upgrade-examples/LoginUpgradeExamples/docs/code.md

```diff
@@ -0,0 +1,12 @@
+## Overview
+
+When converting PatternFly 3 components to PatternFly 4 components, you must also take into consideration the layouts and sizings that PatternFly 3 utilized from Bootstrap 3.
+
+## Usage
+
+| PF3 Class | Applied To | PF4 Class | Applied To | Outcome |
+| -- | -- | -- | -- | -- |
+| `.login-pf` | `<div>` | `.pf-c-login` | `<div>` | Creates a login containing content. |
```

Comment:
`.login-pf` in pf3 applies to `<body>`

Reply:
Class and element tag has been rematched.